app/Resources/views/committee/timeline/_event.html.twig

```diff
@@ -27,10 +27,14 @@
             <li>
                 {% if item.event.finishAt < date() %}
                     Cet événement est terminé
+                {% elseif event.participantsCount < event.capacity %}
```

Comment:
Better to have a dedicated method like `$event.->isFull()` so that the Twig template can be simplified to `{% elseif not event.full %}`.